Gemfile

```diff
@@ -13,8 +13,9 @@ group :development, :test do
   gem 'climate_control'
 end
 
-gem 'azure_mgmt_resources', '~>0.5.0'
-gem 'azure_mgmt_compute',   '~>0.5.0'
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```

Comment:
@vishrutshah I think Github may do this in Github Desktop. Did you check using Github Desktop? Normally you can see the red marker in the PR if there is no end of line.